packages/wmr/src/lib/normalize-options.js

```diff
@@ -82,7 +94,11 @@ export async function normalizeOptions(options, mode) {
 
 			const fileUrl = url.pathToFileURL(configFile);
 			try {
-				custom = await eval('(x => import(x))')(fileUrl);
+				// Node <= 12.18.4 returns an empty module if we append search params to
+				// the URL.
+				const importSource = supportsSearchParams ? `(x => import(x + '?t=${Date.now()}'))` : `(x => import(x))`;
+				// TODO: Do we even need eval here?
```

Comment:
The eval is required in order to prevent Rollup from breaking when it bundles WMR. Without the eval it'll try to bundle the dynamic import.

Reply:
Makes sense, thanks for clearing it up 👍